packages/wrangler/src/__tests__/init.test.ts

```diff
@@ -706,6 +706,7 @@ describe("init", () => {
 							});
 						}
 
+						// @ts-expect-error this is fine
```

Comment:
This an MSW types incompatibility, but I'm pretty sure it should work fine at runtime in tests. I imagine it'll likely be solved by upgrading the MSW version, but I wanted to reduce the complexities in this PR.